release_notes/v1.1.md

```diff
@@ -0,0 +1,334 @@
+# OPEA Release Notes v1.1
+We are pleased to announce the release of OPEA version 1.1, which includes significant contributions from the open-source community. This release addresses over 470 pull requests.
+
+More information about how to get started with OPEA v1.1 can be found at [Getting Started](https://opea-project.github.io/latest/index.html) page. All project source code is maintained in the [repository](https://github.com/opea-project). To pull Docker images, please access the [Docker Hub](https://hub.docker.com/u/opea). For instructions on deploying Helm Charts, please refer to the [guide](https://github.com/opea-project/GenAIInfra/tree/v1.1/helm-charts#readme).
+
+## What's New in OPEA v1.1
+This release introduces more scenarios with general availability, including:
+- Newly supported Generative AI capabilities: [Image-to-Video](https://github.com/opea-project/GenAIComps/pull/465), [Text-to-Image](https://github.com/opea-project/GenAIComps/pull/729), [Text-to-SQL](https://github.com/opea-project/GenAIExamples/tree/main/DBQnA) and [Avatar Animation](https://github.com/opea-project/GenAIComps/pull/775). 
+- [Generative AI Studio](https://github.com/opea-project/GenAIStudio) that offers a no-code alternative to create enterprise Generative AI applications.
+- Expands the portfolio of supported hardware to include [Intel® Arc™ GPUs](https://github.com/opea-project/GenAIComps/pull/641) and [AMD® GPUs](https://github.com/opea-project/GenAIExamples/issues/1153).
+- Enhanced monitoring system, providing real-time insights into runtime status and system resource utilization for [CPU and Intel® Gaudi® AI Accelerator](https://github.com/opea-project/GenAIEval/issues/195), as well as [Horizontal Pod Autoscaling (HPA)](https://github.com/opea-project/GenAIInfra/pull/531).
```

Comment:
```suggestion
- Enhanced monitoring support, providing real-time insights into runtime status and system resource utilization for [CPU and Intel® Gaudi® AI Accelerator](https://github.com/opea-project/GenAIEval/issues/195), as well as [Horizontal Pod Autoscaling (HPA)](https://github.com/opea-project/GenAIInfra/pull/531).
```

Or use the suggestion from Malini, that was already applied once, but somehow lost later?